content/source/docs/enterprise/workspaces/variables.html.md

```diff
@@ -90,6 +90,22 @@ Do not commit a file named `terraform.tfvars` to version control, since TFE will
 
 TFE performs Terraform runs on disposable Linux worker VMs using a POSIX-compatible shell. Before running Terraform, TFE populates the shell with environment variables using the `export` command.
 
+### Special Environment Variables
+
+TFE uses special environment variables to control dangerous or rarely used run behaviors, and reserves several other environment variables for internal use.
+
+The following variables can be set by a user to manage a workspace's run behavior:
+
+- `CONFIRM_DESTROY` — If this environment variable is set to `1` in a workspace, an admin user can destroy all of the infrastructure managed by the workspace using the "Queue destroy plan" button in the settings page. The UI text for the destroy plan button includes a reminder about this safety measure.
+- `TFE_PARALLELISM` — If present, TFE uses this to set `terraform apply`'s `-parallelism=<N>` flag ([more info](/docs/internals/graph.html#walking-the-graph)). Valid values are between 1 and 256, inclusive; the default is `10`. This is rarely necessary, but is available as a workaround for infrastructure providers that error on concurrent operations or implement rate limiting in a non-standard way. We recommend talking to HashiCorp support before using this.
+
+The following variables are reserved for internal use:
+
+- `TFE_TOKEN` and `ATLAS_TOKEN` — Reserved for passing the ephemeral [per-run API token](../run/index.html#backend-details), which is used to read and write state and to access private modules.
```

Comment:
This is both partially incorrect and could maybe be considered sensitive information that we don't want to put on the website? See here for more details about why this is partially incorrect: https://hashicorp.slack.com/archives/C6PE3FELQ/p1540549526006200